samza-core/src/test/java/org/apache/samza/diagnostics/TestDiagnosticsManager.java

```diff
@@ -59,6 +62,7 @@
   private int numPersistentStores = 2;
   private int containerNumCores = 2;
   private boolean autosizingEnabled = false;
+  private Config config = new MapConfig(ImmutableMap.of("job.name", jobName, "job.id", jobId));
```

Comment:
Please add additional configs to make it somewhat representative of the configs that we want to emit.

Reply:
Done.